docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
Aaah, in that way....

docker-compose is missing a `dont_start` or such option for these cases.